apps/rehabstod/src/error/ErrorId.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react'
+import { TooltipIcon } from '../components/TooltipIcon/TooltipIcon'
+
+export function ErrorId({ errorId }: { errorId: string }) {
+  const [displayCopyMessage, setDisplayCopyMessage] = useState(false)
+
+  const handleCopyClick = () => {
+    setDisplayCopyMessage(true)
+    navigator.clipboard.writeText(errorId)
+  }
+
+  return (
+    <>
+      <div className="mt-5 flex">
+        <p className="mr-2 font-bold">FEL-ID:</p>
+        {errorId}
+        <TooltipIcon
+          description="Kopiera fel-id"
+          name="copy-link"
```

Comment:
After speaking with UX, we have decided that the tooltip icon should be updated to ```name="copy-file"```